open_spiel/games/phantom_go/phantom_go_board.cc

```diff
@@ -481,9 +419,45 @@ void GoBoard::SetStone(VirtualPoint p, GoColor c) {
   board_[p].color = c;
 }
 
+std::array<GoColor, kMaxBoardSize* kMaxBoardSize> PhantomGoBoard::getObservationByID(int player_id) const
+{
+    return observations[player_id];
+}
+
+std::string PhantomGoBoard::observationToString() const
+{
+    std::stringstream ss;
+    ss << "\nObservation white:\n";
+
+    for (int x = board_size_ - 1; x >= 0; x--)
```

Comment:
Please use google code style, you can set up in clion as https://blog.jetbrains.com/clion/2015/07/new-clion-1-1-eap-select-and-use-your-favourite-code-style/

Reply:
This should reformat
```
for(...) 
{
  ...
}
```
to
```
for(...) {
  ...
}
```